shacl-shacl/shsh12-union.ttl

```diff
@@ -0,0 +1,19 @@
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX prof: <http://www.w3.org/ns/dx/prof/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+
+
+sh:ShaclShaclUnion
+    a owl:Ontology ;
+    prof:isProfileOf
+        sh:ShaclShaclCore ,
+        sh:ShaclShaclCompactSyntax ,
+        sh:ShaclShaclInferenceRules ,
+        sh:ShaclShaclNodeExpressions ,
+        sh:ShaclShaclProfiling ,
+        sh:ShaclShaclSparql ,
+        sh:ShaclShaclUserInterface ;
```

Comment:
This is the only part I'm not sure about.

SHACL-Core already has practice defined for `owl:imports`, which itself has operational rules for transitive importing.  If using `prof:isProfileOf` here, are we inducing a whole new rule set that is similar to `owl:imports`' rules?  If SHACL-SHACL is used as an operational example for people to start writing their own SHACL graphs, why should they choose `prof:isProfileOf` rather than `owl:imports`?

Actually, probably relatedly: Why are these IRIs in the SHACL namespace, and not the `shsh:` namespace?

Why would we introduce `prof:isProfileOf` here?